tower-http/src/content_encoding.rs

```diff
@@ -1,42 +1,45 @@
 pub(crate) trait SupportedEncodings: Copy {
+    fn br(&self) -> bool;
     fn gzip(&self) -> bool;
     fn deflate(&self) -> bool;
-    fn br(&self) -> bool;
 }
 
-#[derive(Copy, Clone, Debug, PartialEq, Eq)]
+// The order of the variants is important: If the client has equal highest preference for multiple
+// encodings, the first of these is chosen. If the client prefers the `*` encoding, the first
+// variant not explicitly mentioned by the client is chosen.
```

Comment:
Where did you find what the order should be here (after identity)? I couldn't find it in the specification when skimming it where it stated that Br for example should have higher prio than gzip if both have equal q values